src/components/FeedbackBox/FeedbackBox.tsx

```diff
@@ -0,0 +1,477 @@
+                            type="submit"
+                          >
+                            Submit
+                          </Button>
+                        </ButtonGroup>
+                      </FormField>
+                    </>
+                  )}
+
+                  {/* Confirmation Screen */}
+                  {isConfirmationView && (
+                    <>
+                      <Box
+                        key="confirmationWrapper"
+                        margin="auto"
+                        tabIndex={0}
+                        textAlign="center"
+                        ref={focusRef}
+                      >
+                        <Icon name="actionCheckCircleFilled" size="large" />
+                        <Text fontWeight="medium">
+                          Thank you for submitting your feedback.
+                        </Text>
+                        {showEmailField && (
+                          <Text>
+                            If you provided an email address and require a
+                            response, our service staff will reach out to you
+                            via email.
+                          </Text>
+                        )}
+                        {confirmationText ? (
+                          <Text>{confirmationText}</Text>
+                        ) : undefined}
+                      </Box>
+                      {privacyPolicyField}
+                      <FormField>
+                        <ButtonGroup buttonWidth="full" id="submit-cancel">
+                          <Button
+                            id="return-browsing"
+                            buttonType="secondary"
+                            onClick={closeAndResetForm}
+                          >
+                            Return to Browsing
+                          </Button>
+                        </ButtonGroup>
+                      </FormField>
+                    </>
+                  )}
+
+                  {/* Error Screen */}
+                  {isErrorView && (
+                    <>
+                      <Box
+                        color="ui.error.primary"
+                        key="errorWrapper"
+                        margin="auto"
+                        tabIndex={0}
+                        textAlign="center"
+                        ref={focusRef}
+                      >
+                        <Icon
+                          color="ui.error.primary"
+                          name="errorFilled"
+                          size="large"
+                        />
+                        <Text fontWeight="medium">
+                          Oops! Something went wrong. An error occured while
+                          processing your feedback.
+                        </Text>
+                      </Box>
+                      {privacyPolicyField}
+                      <FormField>
+                        <ButtonGroup buttonWidth="full" id="submit-cancel">
```

Comment:
Sorry, I neglected to add this comment in the previous review.  These should also be flipped.